### PR TITLE
Scribe of bl add 24995

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: new/PRS13411GondareH.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13411GondareH" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Gondare Habtu</title>
```
Comment (DenisNosnitsin1970) [inline new/PRS13411GondareH.xml:8]: Ist he ጎንዳሬ or ጎንደሬ?

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/PRS13411GondareH.xml (hunk 1 above)
Comment (DariaElagina) [reply new/PRS13411GondareH.xml:8]: I can judge only by the description of Wright, he gives: ጎንደሬ

**Review by DenisNosnitsin1970 — APPROVED** [review]
Gondare is most probably a "nisba", identifying him as Gondarian; 
it is most probably not a name.

File: new/PRS13411GondareH.xml [hunk 2]
```diff
@@ -45,17 +45,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">Gondare Habtu</persName>
+                    <persName xml:lang="gez" xml:id="n1">ጎንደሬ፡ ሀብቱ፡</persName>
```
Comment (eu-genia) [inline new/PRS13411GondareH.xml:48]: maybe Gondare should be better be marked up as addName...

**Review by eu-genia — APPROVED** [review]

File: new/PRS13411GondareH.xml (hunk 2 above)
Comment (DariaElagina) [reply new/PRS13411GondareH.xml]: Like that?

**Review by eu-genia — APPROVED** [review]

**Pull request merged**